Mathlib/Order/Interval/Set/Union.lean

```diff
@@ -0,0 +1,34 @@
+/-
+Copyright (c) 2025 Oliver Butterley. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Oliver Butterley
+-/
+import Mathlib.Data.Set.Lattice
+import Mathlib.Data.Finset.Range
+
+/-!
+# Extra lemmas about unions of intervals
+
+This file contains lemmas about finite unions of intervals which can't be inclued with the lemmas
+concerning infinite unions in `Order.Interval.Set.Disjoint` because we use `Finset.range`.
+-/
+
+open Set
+
+/-- A union of intervals contains the interval defined by choosing any two of the end points. -/
```

Comment:
```suggestion
/-- A union of consecutive intervals contains the interval defined by the initial and final points. -/
```
I am not sure what `any two of the end points` is supposed to mean. Same thing below.